templates/observability/index.html

```diff
@@ -0,0 +1,519 @@
+{% extends "observability/base_observability.html" %}
+
+{% block title %}Open source observability for enterprises{% endblock %}
+
+{% block meta_description %}Enterprise-class open source observability with long-term support for Prometheus, Alertmanager, Grafana and more.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row">
+      <div class="col-7">
+        <h1>Open source observability for enterprises</h1>
+        <h2>Observability with open source monitoring tools and dashboards, secured and supported</h2>
+        <p>Metrics, distributed tracing, logs, dashboards, alerts. Always know how well your applications are running in the Cloud, in containers, on virtual machines or bare metal.</p>
+        <p>Canonical offers security and support for the best in class open source monitoring tools and dashboards, and can also run these tools for you on private and public clouds.</p>
+        <p><a href="/observability/contact-us" class="p-button--positive js-invoke-modal">Contact Us</a></p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/e68fd355-1+discovery_AW.svg",
+          alt="",
+          width="403",
+          height="230",
+          hi_def=True,
+          loading="lazy"
```

Comment:
This image is "above the fold", so this value shoud be `auto` or `eager`, to reduce the chance of layout shift when the image loads. See `image-template` [docs](https://github.com/canonical-web-and-design/canonicalwebteam.image-template#parameters)